src/ResourceManager/Network/Commands.Network/ServiceEndpointPolicy/ServiceEndpointPolicyDefinition/AddAzureServiceEndpointPolicyDefinitionCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
@chandrasekarsrinivasan 
help file missing for this command,

"Assembly","HelpFile","Target","Severity","ProblemId","Description","Remediation"
"","","Microsoft.Azure.Commands.Network.AddAzureServiceEndpointPolicyDefinitionCommand","1","6050","Help missing for cmdlet Add-AzureRmServiceEndpointPolicyDefinition implemented by class Microsoft.Azure.Commands.Network.AddAzureServiceEndpointPolicyDefinitionCommand","Add Help record for cmdlet Add-AzureRmServiceEndpointPolicyDefinition to help file."